pkg/plugins/diff/delta_diff_server/src/utils.rs

```diff
@@ -31,48 +30,39 @@ const TIMESTAMP_KEY: &str = "timestamp";
 
 pub(crate) fn construct_storage_config(config: GatewayConfig) -> HashMap<String, String> {
     let mut s3_config: HashMap<String, String> = HashMap::new();
```

Comment:
Not in this PR, but it seems odd to have a mutable map for a static map.  Can't we just set up an immutable map here?

Reply:
changed it
thanks!